buildSrc/src/main/kotlin/BuildInfo.kt

```diff
@@ -73,6 +83,16 @@ open class BuildInfo(project: Project) {
     project.extensions.getByType<VersionCatalogsExtension>().named("libs")
   }
 
+  val zig: Zig = Zig()
+
+  inner class Zig {
+    val version: String get() = libs.findVersion("zig").get().toString()
+
+    val installDir: Path get() = project.projectDir.toPath().resolve(".gradle/zig/zig-$version")
```

Comment:
I followed the node-gradle plugin's lead and put the Zig distribution in `.gradle`. Not sure if this is the correct thing though.